src/OpenTelemetry.Instrumentation.SqlClient/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* [Breaking] Enrich event name`OnCustom` is changed to `OnEventWritten`.
```

Comment:
This is for consistency. Since we are not using `OnCustom` anywhere now. I am ok with changing it back to `OnCustom` if we don't want the breaking change.